scripts/check-RN-link-perms.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# Checks if a release notes file is updated. If true, searches the release notes file for links to bugs that are behind a login and fails the build.
+# We should not include internal bugs in our release notes.
+
+# Get the repo path and files changed
+REPO_PATH="$(git rev-parse --show-toplevel)"
+FILES=$(git diff --name-only HEAD~1 HEAD --diff-filter=d "*.adoc" ':(exclude)_unused_topics/*')
```

Comment:
Do the release note file finding in this line. there is no need for the while loop below

Reply:
`git diff --name-only HEAD~1 HEAD --diff-filter=d "release_notes/*-release-notes.adoc"`

should work - or similar.

Reply:
Great, thanks. Maybe `*release-notes*` to capture microshift, rosa etc. release notes